front-end/src/components/Body/PostsPage/PostPage.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect } from "react";
+import "./style.scss";
+import Card from "./Card/Card";
```

Comment:
To import quickly use can use rename Card.js to index.js and import as below

```JS
import Card from "./Card";
```
<img width="984" alt="Screen Shot 2022-03-10 at 18 13 18" src="https://user-images.githubusercontent.com/86216577/157650628-9e28d5db-dd09-4cff-9380-a05ae2737706.png">
Thanks